src/models/adr-details.ts

```diff
@@ -9,24 +10,101 @@ export interface AdrDetails {
   brakeDeclarationsSeen?: boolean;
   brakeDeclarationIssuer?: string;
   brakeEndurance?: boolean;
-  weight?: string;
-  compatibilityGroupJ?: boolean;
+  weight?: number;
+  newCertificateRequested?: boolean;
+  compatibilityGroupJ?: compatibilityGroupJEnum;
   documents?: string[];
-  permittedDangerousGoods?: string[];
-  additionalExaminerNotes?: string;
+  permittedDangerousGoods?: permittedDangerousGoodsEnum[];
```

Comment:
ADR documentation states that this **_shouldn't_** be an enum in the backend due to the historic data containing erroneous values